pkg/syndesis/action/action.go

```diff
@@ -41,3 +49,34 @@ func setNamespaceAndOwnerReference(resource interface{}, syndesis *v1alpha1.Synd
 		})
 	}
 }
+
+func createOrReplace(res runtime.Object) error {
+	return createOrReplaceForce(res, false)
+}
+
+func createOrReplaceForce(res runtime.Object, force bool) error {
+	if err := sdk.Create(res); err != nil && k8serrors.IsAlreadyExists(err) {
+		if force || canResourceBeReplaced(res) {
+			err = sdk.Delete(res, sdk.WithDeleteOptions(&metav1.DeleteOptions{}))
+			if err != nil {
+				return err
+			}
+			return sdk.Create(res)
+		} else {
+			return nil
+		}
+	} else {
+		return err
+	}
+}
+
+func canResourceBeReplaced(res runtime.Object) bool {
+	if !replaceResourcesIfPresent {
```

Comment:
Not sure about this flag as this is always set to `true` ?

Reply:
I've left it as "constant" because we've not defined operator configuration yet..